shunting-yard-common/src/main/java/com/hotels/shunting/yard/common/event/apiary/SerializableApiaryInsertTableEvent.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (C) 2016-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.shunting.yard.common.event.apiary;
+
+import java.util.List;
+import java.util.Map;
+
+import org.apache.hadoop.hive.metastore.events.InsertEvent;
+
+import com.hotels.shunting.yard.common.event.SerializableListenerEvent;
+
+public class SerializableApiaryInsertTableEvent extends SerializableListenerEvent {
+  private static final long serialVersionUID = 1L;
+
+  /**
+   * This class deliberately extends {@link SerializableListenerEvent} and not {@link SerializableApiaryListenerEvent}
```

Comment:
is this some old coment there is no `sourceMetastoreUris` in the `SerializableApiaryListenerEvent`
Actually `SerializableApiaryListenerEvent` seems not necessary?

Reply:
:) This also got removed as part of code deletion.